docs/src/examples/stadium.md

```diff
@@ -0,0 +1,137 @@
+# Driving in a Stadium
+
+This example demonstrates a 2D driving simulation where cars drive around a three-lane stadium.
+The entities are defined by the types:
+
+- `S` - `VehicleState`, containing the vehicle position (both globally and relative to the lane) and speed 
+- `D` - `VehicleDef`, containing length, width, and class
+- `I` - `Symbol`, a unique label for each vehicle
+
+The environment is represented by a `Roadway` object which 
+allows to define roads consisting of multiple lanes based on the RNDF format.
+
+We load relevant modules and generate a 3-lane stadium roadway:
+
+```@example driving_in_circles
+using AutomotiveDrivingModels
+using AutoViz
+using Distributions
+
+roadway = gen_stadium_roadway(3)
+snapshot = render([roadway], camera=FitToContentCamera(.1))
+write("stadium.svg", snapshot) # hide
+```
+![three lane stadium](stadium.svg)
+
+As a next step, let's populate a scene with vehicles
+
+```@example driving_in_circles
+w = DEFAULT_LANE_WIDTH
+scene = Frame([
+    Entity(VehicleState(VecSE2(10.0,  -w, 0.0), roadway, 29.0), VehicleDef(), :alice),
+    Entity(VehicleState(VecSE2(40.0, 0.0, 0.0), roadway, 22.0), VehicleDef(), :bob),
+    Entity(VehicleState(VecSE2(30.0, -2w, 0.0), roadway, 27.0), VehicleDef(), :charlie),
+])
+car_colors = get_pastel_car_colors(scene)
+renderables = [
+    roadway,
+    (FancyCar(car=veh, color=car_colors[veh.id]) for veh in scene)...
+]
+snapshot = render(renderables, camera=FitToContentCamera(.1))
+write("stadium_with_cars.svg", snapshot) # hide
+```
+![stadium with cars](stadium_with_cars.svg)
+
+We can assign driver models to each agent and simulate the scenario.
+
+```@example driving_in_circles
+timestep = 0.1
+nticks = 300
+
+models = Dict{Symbol, DriverModel}(
+    :alice => LatLonSeparableDriver( # produces LatLonAccels
+        ProportionalLaneTracker(), # lateral model
+        IntelligentDriverModel(), # longitudinal model
+    ),
+    # :bob => Tim2DDriver(
+    #     timestep, mlane = MOBIL(timestep),
+    # ),
+    :bob => StaticDriver{AccelTurnrate, MvNormal}(
+        MvNormal([0.0,0.0], [1.0,0.1])
+    ),
+    :charlie => StaticDriver{AccelTurnrate, MvNormal}(
```

Comment:
what is going on with Charlie? leaves the track (see gif). Probly because this is a static driver, but should explain more (or maybe even use a different driver model, this behavior can be quite confusing)

Reply:
It is because it is a `StaticDriver`, It might be nice to showcase the different action types to have some that follow the lane and some that do not, but I agree that a sentence explaining that it is not a bug would be useful.